.readme-partials.yml

```diff
@@ -132,3 +132,24 @@ body: |-
     [LOGGING_TRACE_KEY]: 'custom-trace-value'
   }, 'Bunyan log entry with custom trace field');
   ```
+
+  ### Error handling with a default callback
+  The `LoggingBunyan` class creates an instance of `Logging` which creates the `Log` class from `@google-cloud/logging` package to write log entries. 
+  The `Log` class writes logs asynchronously and there are cases when log entries cannot be written and an error is 
+  thrown - if error is not handled properly, it could crash the application. One possible way to handle the error is to provide a default callback
```

Comment:
```suggestion
  thrown.  If the error is not handled properly, it could crash the application. One possible way to handle the error is to provide a default callback
```